pom.xml

```diff
@@ -9,7 +9,7 @@
     <properties>
         <java.version>11</java.version>
         <revision>0-DEFAULT</revision>
-        <wma.maven.url>https://cida.usgs.gov/artifactory</wma.maven.url>
+        <wma.maven.url>https://artifactory.wma.chs.usgs.gov/artifactory</wma.maven.url>
```

Comment:
Do we want to hard-code this URL, or would it make sense to examine making it a variable that is injected at build time?  (we tend to be fond of changing our URLS...)

Reply:
I agree we could parameterize this url and pull it in via serverless pulling from secrets manager | parameter store or something similar.  However, I think it would be good to do this after we've migrated the java lambdas to chs artifactory so we can get up and running on active development as soon as possible.

Reply:
This is in the Java pom. To place it anywhere else would mean having a maven profile setup in every place the code is compiled. In other words, every developers environment, travis, and jenkins.

Reply:
In that case, it probably doesn't make to put it anywhere else. It's only worthwhile if we have to change it in fewer places.

Reply:
Additionally, I am in concurrence that we should refrain from expanding this activity to include improvements beyond addressing the problems that prevent us building and deploying at present. One battle at a time.